<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtDemo</class>
 <widget class="QWidget" name="QtDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtDemo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <item>
    <layout class="QHBoxLayout" name="Layout_Title" stretch="0,1,0,0,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>应用标题</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_min">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_max">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_close">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>程序主体内容</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
